.devcontainer/Dockerfile

```diff
@@ -116,5 +117,7 @@ RUN chmod 644 /usr/share/grafana/conf/provisioning/datasources/psinode-datasourc
 ADD grafana/dashboards /var/lib/grafana/dashboards
 COPY grafana/grafana-psinode.ini /etc/grafana/grafana-psinode.ini
 
+COPY gdb/.gdbinit /root/.gdbinit

```

Comment:
Another solution would be to put the commands in `launch.json` (in `setupCommands`). That would allow referencing `${workspaceFolder}`.